deploy/fabfile.py

```diff
@@ -33,26 +33,46 @@
 home = "/home/site"
 log = "%s/hookshot.log" % home
 current = "%s/%s/current" % (home, environment)
+ruby = "/opt/install/rbenv/shims/ruby"
 
 # principal command to run upon update
-command = "cd %s && git pull && bundle exec jekyll build >> %s" % (current, log)
+command = "cd %s && %s go server_build >> %s" % (current, ruby, log)
 
 ## can be run on their own
 
 def start():
-  run(
-    "cd %s && forever start -l %s -a deploy/hookshot.js -p %i -b %s -c \"%s\""
-    % (current, log, port, environment, command)
-  )
+  with cd(current):
+    run(
+      "forever start -l %s -a deploy/hookshot.js -p %i -b %s -c \"%s\""
+      % (log, port, environment, command)
+    )
 
 def stop():
-  run(
-    "cd %s && forever stop deploy/hookshot.js -p %i -b %s -c \"%s\""
-    % (current, port, environment, command)
-  )
+  with cd(current):
+    run(
+      "forever stop deploy/hookshot.js -p %i -b %s -c \"%s\""
+      % (port, environment, command)
+    )
 
 def restart():
-  run(
-    "cd %s && forever restart deploy/hookshot.js -p %i -b %s -c \"%s\""
-    % (current, port, environment, command)
-  )
+  with cd(current):
+    run(
+      "forever restart deploy/hookshot.js -p %i -b %s -c \"%s\""
+      % (port, environment, command)
+    )
+
+def update_data():
+  with cd(current):
+    run("%s go update_submods" % (ruby))
+    run("%s go update_data" % (ruby))
+
+def update_submods():
+  with cd(current):
+    run("%s go update_submods" % (ruby))
+
+def build(scope = 'simple'):
+  with cd(current):
+    if scope = 'full'
+      run("%s go server_build" % (ruby))
+    elif scope = 'simple'
```

Comment:
Not a blocker, but consider this instead:

```python
def build(scope = ''):
  with cd(current):
    if scope == 'full':
      run("%s go server_build" % ruby)
    else:
      run('%s go build' % ruby)
```

Not only does eliminate the extra conditional (which should've been `==`, btw), we eliminate the dreaded `s`-word. :-)